spec/database_cleaner/active_record/base_spec.rb

```diff
@@ -1,59 +1,61 @@
-require 'spec_helper'
 require 'active_record'
 require 'database_cleaner/active_record/base'
 require 'database_cleaner/shared_strategy'
+require 'support/active_record/schema_setup'
 
 class FakeModel
   def self.connection
     :fake_connection
   end
 end
 
-module DatabaseCleaner
-  describe ActiveRecord do
-    it { should respond_to(:available_strategies) }
-
-    describe "config_file_location" do
-      subject { ActiveRecord.config_file_location }
+describe DatabaseCleaner::ActiveRecord do
```

Comment:
@botandrose I think the latest convention is to start with `RSpec.describe` instead of `describe` - Could you make it so?

Reply:
I have set the configuration setting to disable RSpec's monkey patching, entirely, so this convention is enforced, now.